scripts/xtp_autogen_mapping.in

```diff
@@ -472,15 +473,18 @@ def random_selection(fragment) -> np.ndarray:
 def angle_fragment(idx: np.ndarray, fragment: Chem.rdchem.Mol) -> float:
     try:
         conformer = fragment.GetConformer()
-        pos_1, pos_2, pos_3 = [
-            conformer.GetAtomPosition(idx[x].item()) for x in range(3)]
-        vec1 = np.array([pos_1.x - pos_2.x, pos_1.y -
-                         pos_2.y, pos_1.z - pos_2.z])
-        vec2 = np.array([pos_1.x - pos_3.x, pos_1.y -
-                         pos_3.y, pos_1.z - pos_3.z])
+        if len(idx)>1:
```

Comment:
does this not also break for the case of two atoms?

Reply:
good spot. It should be if len(idx)>2 indeed